csrc/host_ir/executor.h

```diff
@@ -165,6 +166,7 @@ class HostIrEvaluator final : public OptOutDispatch {
   }
 
   std::unique_ptr<HostIrContainer> container_;
+  std::unique_ptr<HostIrJit> jit_;
```

Comment:
HostIrJit is only compiled when we enable it.
So this won't compile when we not opt-in for HostIrJit.